### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by bossmonkey98**
Doc page added
Apex UI — Mozilla Firefox 17-04-2022 12_21_12
Apex UI — Mozilla Firefox 17-04-2022 12_21_20

### CODE REVIEW

File: docs.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    grid-area: conten;
+}
+
+/* code snippet */
+.code-snippet{
+    margin: 10px;
+    padding: 10px;
+    border: 1px solid var(--bg-color);
+    border-radius: 10px;
+    background-color: var(--ternary);
```
Comment (arjun-computer-geek) [inline docs.css:42]: This is great to use css variable

**Review by arjun-computer-geek — APPROVED** [review]
Good class names. Goog job

File: docs.css (hunk 1 above)
Comment (t007rushi) [reply docs.css:42]: @arjun-computer-geek rightly pointed out

File: index.html [hunk 2]
```diff
@@ -1,11 +1,32 @@
...
+        <div id="logo">
+            <a href="index.html">
+                <span><img class="logo-icon" src="./assets/logo/apex-legends-symbol-white.png" alt="A"></span><span class="name">pex UI</span></a>
+        </div>
+        <div class="Home">
+            <a href="https://github.com/bossmonkey98"><i class="bi bi-github"></i></a>
+            <a href="https://www.linkedin.com/in/arun-kumar-0b95911aa/"><i class="bi bi-linkedin"></i></a>
+        </div>
+    </header>
+    <div class="content-container">
```
Comment (t007rushi) [inline index.html:22]: be consistent with naming as...navbar-content

**Review by t007rushi — APPROVED** [review]
go ahead

**Pull request merged**